pkg/nsx/services/securitypolicy/builder.go

```diff
@@ -69,10 +73,10 @@ func (service *SecurityPolicyService) buildSecurityPolicy(obj *v1alpha1.Security
 
 	for ruleIdx, rule := range obj.Spec.Rules {
 		// A rule containing named port may expand to multiple rules if the name maps to multiple port numbers.
-		expandRules, ruleGroups, err := service.buildRuleAndGroups(obj, &rule, ruleIdx)
+		expandRules, ruleGroups, shares, err := service.buildRuleAndGroups(obj, &rule, ruleIdx)
 		if err != nil {
 			log.Error(err, "failed to build rule and groups", "rule", rule, "ruleIndex", ruleIdx)
```

Comment:
Add shares in log？

Reply:
we add share log in pkg/nsx/services/securitypolicy/builder.go#L782